kernel/kernel-api/src/main/java/io/delta/kernel/internal/ScanImpl.java

```diff
@@ -105,6 +105,18 @@ public CloseableIterator<FilteredColumnarBatch> getScanFiles(Engine engine) {
     return getScanFiles(engine, false);
   }
 
+  /**
+   * Get an iterator of data files in this version of scan that survived the predicate pruning.
+   *
+   * @param engine the {@link Engine} instance to use
+   * @param includeStats whether to read and include the JSON statistics
+   * @return data in {@link ColumnarBatch} batch format. Each row correspond to one survived file.
+   */
+  public CloseableIterator<FilteredColumnarBatch> getScanFiles(
+      Engine engine, boolean includeStats) {
+    return getScanFiles(engine, includeStats, false);
```

Comment:
nit: do we need this pass-through method today? can't we just call `getScanFiles(engine, includeStats, false);` directly?